clar2wasm/src/wasm_generator.rs

```diff
@@ -1716,7 +1716,7 @@ impl WasmGenerator {
                     builder.call(
                         self.module
                             .funcs
-                            .by_name("stdlib.chain_id")
+                            .by_name("chain_id")
```

Comment:
Sames as above. `stdlib.` prefix is now part of function present on Clarity standard library.